models/biswap/bnb/biswap_bnb_schema.yml

```diff
@@ -20,84 +20,57 @@ models:
             - version
             - tx_hash
             - evt_index
-            - trace_address
-      - check_dex_seed_legacy:
+      - check_dex_seed:
           blockchain: bnb
           project: biswap
-          version: 1
+          version: 2
     columns:
-      - &blockchain
```

Comment:
why remove `&` usage? if you keep, then you can reuse with `*` in following spells added to schema to avoid clutter

Reply:
Sorry didn't know this would cause clutter, fixed now